packages/template-retail-react-app/app/components/_app/index.jsx

```diff
@@ -227,6 +234,12 @@ const App = (props) => {
         })
     }, [])
 
+    const {usid} = useUsid()
+    const createShopperContext = useShopperContextsMutation('createShopperContext')
+    const deleteShopperContext = useShopperContextsMutation('deleteShopperContext')
+    const updateShopperContext = useShopperContextsMutation('updateShopperContext')
+    useShopperContextSearchParams(site.id)
```

Comment:
Do we need to pass siteId as argument? Can we call inside useShopperContext hook?
```
 const {site} = useMultiSite()
```